build.gradle

```diff
@@ -23,13 +23,18 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3:3.12.0' // Apache 2.0
 
     // GeoTools (LGPL 2.1)
-    implementation 'org.geotools:geotools:21.0'
-    implementation 'org.geotools:gt-epsg-hsql:21.0'
-    implementation 'org.geotools:gt-opengis:21.0'
-    implementation 'org.geotools:gt-geometry:21.0'
-    implementation 'org.geotools:gt-referencing:21.0'
+    implementation 'org.geotools:geotools:29.1'
+    implementation 'org.geotools:gt-epsg-hsql:29.1'
+    implementation 'org.geotools:gt-opengis:29.1'
+    implementation 'org.geotools:gt-geometry:24.2'
```

Comment:
Any specific reason why this library uses a different version than the others?
I would expect that this could cause problems between the different Geotools libraries.

Reply:
24.2 and 24.4.01 seems to be the latest version of gt-geometry
https://mvnrepository.com/artifact/org.geotools/gt-geometry
I was also surprised to see that there is such a bit difference between the versions.

Reply:
So maybe this module was dropped or merged into another one?
What happens if you remove it?

Reply:
I have removed it and change the 'org.geotools:geotools:29.1' to 'org.geotools:gt-main:29.1' which is the core module of GeoTools contains fundamental GIS functionality, including geometry-related classes.